backend/src/main/java/de/bund/digitalservice/ris/norms/application/service/TimeMachineService.java

```diff
@@ -120,23 +123,37 @@ public Norm applyPassiveModifications(ApplyPassiveModificationsUseCase.Query que
                 return;
               }
 
-              final var nodeToChange =
-                  NodeParser.getNodeFromExpression(
-                      String.format("//*[text()[contains(.,'%s')]]", mod.getOldText().get()),
-                      targetNode.get());
-
-              if (nodeToChange.isEmpty()) {
-                return;
-              }
-
-              final var modifiedTextContent =
-                  nodeToChange
-                      .get()
-                      .getTextContent()
-                      .replaceFirst(mod.getOldText().get(), mod.getNewContent().get());
-              nodeToChange.get().setTextContent(modifiedTextContent);
+              if (mod.usesQuotedText()) applyQuotedText(mod, targetNode.get());
+              if (mod.usesQuotedStructure()) applyQuotedStructure(mod, targetNode.get());
             });
 
     return norm;
   }
+
+  private void applyQuotedText(Mod mod, Node targetNode) {
+    if (mod.getOldText().isEmpty() || mod.getNewText().isEmpty()) return;
+    String oldText = mod.getOldText().get();
+    String newText = mod.getNewText().get();
+
+    String xPathOldText = String.format("//*[text()[contains(.,'%s')]]", oldText);
+    final Node nodeToChange = NodeParser.getMandatoryNodeFromExpression(xPathOldText, targetNode);
+
+    final String modifiedTextContent = nodeToChange.getTextContent().replaceFirst(oldText, newText);
+    nodeToChange.setTextContent(modifiedTextContent);
+  }
+
+  private void applyQuotedStructure(Mod mod, Node targetQuotedStructure) {
```

Comment:
This logic is wrong. We need to replace the target node, not append the new node to that one.